apps/wallet/src/ui/app/redux/slices/account/index.ts

```diff
@@ -115,7 +111,10 @@ const accountSlice = createSlice({
         ) => {
             state.isLocked = payload.isLocked;
             state.isInitialized = payload.isInitialized;
-            state.address = payload.activeAddress || null; // is already normalized
+            state.serializedInfo =
```

Comment:
why do we need to update this and not just move the find functionality to the accountSelector? It seems weird to have to duplicate the account info here we already have them in accountsAdapter right?

Reply:
maybe, I suck at Redux 😆. Will take a second look at this

Reply:
Okay you're right, this was redundant and kind of weird since accounts are already a Redux entity in state